docker/.env.docker.dev

```diff
@@ -14,3 +14,4 @@ UPWARD_JS_UPWARD_PATH=upward.yml
 UPWARD_JS_BIND_LOCAL=1
 UPWARD_JS_LOG_URL=1
 CHECKOUT_BRAINTREE_TOKEN=sandbox_8yrzsvtm_s2bg8fs563crhqzk
+GOOGLE_MAPS_API_KEY=redacted
```

Comment:
Do you have a sandbox dev token?

Reply:
Google Dev Console requires you to add a payment method to the API key now. So while we do maintain a few within the team we recommend each team to maintain their own, otherwise we'd end up with a large bill on someones card. 

_This is annoying and we contacted Google about getting a sandbox key, but they said no._

Reply:
Understandable lol